<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="glass" name="GLASS" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>861</width>
      <height>431</height>
     </rect>
    </property>
    <property name="rows" stdset="0">
     <string>100</string>
    </property>
    <property name="colomns" stdset="0">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во точек</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Radius_Edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ChangeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Y:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;X:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="P_X_MIN">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>600.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="P_Y_MIN">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>600.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Минимум</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Y:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;X:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="P_X_MAX">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>600.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="P_Y_MAX">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>600.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Максимум</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_5</zorder>
   <zorder>GLASS</zorder>
   <zorder>label_13</zorder>
   <zorder>Radius_Edit</zorder>
   <zorder>frame_6</zorder>
   <zorder>frame_7</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>glass</class>
   <extends>QWidget</extends>
   <header>glass.h</header>
   <container>1</container>
   <slots>
    <signal>signalDiagShow(int)</signal>
    <slot>slotDiagShow(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ChangeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotUpdateButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotUpdateButton()</slot>
  <slot>slotColor()</slot>
 </slots>
</ui>
